src/font.ts

```diff
@@ -347,13 +363,8 @@ export class Font {
   // Support distributions of the typescript compiler that do not yet include the FontFace API declarations.
   // eslint-disable-next-line
   // @ts-ignore
-  static async loadWebFont(fontName: string, woffURL: string, includeWoff2: boolean = true): Promise<FontFace> {
-    const woff2URL = includeWoff2 ? `url(${woffURL}2) format('woff2'), ` : '';
-    const woff1URL = `url(${woffURL}) format('woff')`;
-    const woffURLs = woff2URL + woff1URL;
-    // eslint-disable-next-line
-    // @ts-ignore
-    const fontFace = new FontFace(fontName, woffURLs);
+  static async loadWebFont(fontName: string, woffURL: string): Promise<FontFace> {
+    const fontFace = new FontFace(fontName, `url(${woffURL})`);
```

Comment:
If you look at the original version of loadWebFont, it specifies the woff2URL first, and then the woff1URL. The constructor accepts fallback fonts: https://stackoverflow.com/questions/41061599/how-to-load-fallback-font-formats-with-fontface-api

The browser will look for the woff2 file first, and if it returns a 404, it will fall back to the woff file. So this shouldn't have an impact on network traffic if the woff2 file exists and the user's browser supports woff2.

Anyways, I am OK if you change the API here since the original isn't the clearest. 

However, it would be nice if there was a way to support the fallback approach. The FontFace constructor does not require the url to point to a woff file. In fact, I've verified that you can pass in an OTF and my version of Chrome loads the OTF perfectly fine. :-)

Maybe the method signature here should be:

```
static async loadWebFont(fontName: string, fontURLs: string | string[]): Promise<FontFace> {
```

If we pass in a single string, then do what you are doing now. 

If we pass in a string[], then build up a comma separated list of fallback URLs?

Reply:
What about providing the name without extension?

```
  static WEB_FONT_FILES: Record<string /* fontName */, string /* fontPath */> = {
    Bravura: 'bravura/Bravura_1.392',
    BravuraText: 'bravura/BravuraText_1.393',
...
 ```

and then adding them in the function.

Reply:
OK... I threw together (an incomplete and untested) approach for `loadWebFont()`

```ts
static async loadWebFont(fontFamily: string, fontURLs: string | string[], localNames?: string | string[]) {
    // Compose a src string that looks like:
    // local('FontName'), local('FontName-Bold'), url('font.woff2') format('woff2'), url('font.woff') format('woff')
    let sources: string[] = [];

    if (localNames) {
        if (typeof localNames === "string") {
            sources.push(`local('${localNames}')`);
        } else {
            for (const n of localNames) {
                sources.push(`local('${n}')`);
            }
        }
    }

    if (typeof fontURLs === "string") {
        fontURLs = [fontURLs];
    }

    for (const fontURL of fontURLs) {
        // Determine the type based on the extension.
        // .otf => opentype
        // .ttf => truetype
        // .woff => woff
        // .woff2 => woff2
        const extension = fontURL.split(".").pop();
        let fontType = "";
        switch (extension) {
            case "otf":
                fontType = "opentype";
                break;
            case "ttf":
                fontType = "truetype";
                break;
            case "woff":
                fontType = "woff";
                break;
            case "woff2":
            default:
                fontType = "woff2";
                break;
        }

        sources.push(`url('${fontURL}') format('${fontType}')`);
    }

    const src = sources.join(", ");
    console.log(src);
    // ... DO THE REST HERE
    // new FontFace(fontFamily, src).....
}

```

Reply:
This new approach allows developers to load ONE or MORE fonts. The font types will be automatically detected by file extension.

Additionally, the developer can specify a local name (or more than one) so that the user agent can first look on the local system and load that font (without making a network request).

Reply:
I suggest tha you make a PR for this one